### ISSUE TITLE
okex v1 problem

### ISSUE THREAD

**Issue by JeremyChiaYilin** [ccxt/ccxt#6116]
I use okex v1 api, everything was ok before.

But today,  it response the error 
 'ccxt.base.errors.ExchangeNotAvailable: POST https://www.okex.com/v2/futures/pc/market/futuresCoin 404 Not Found {"code":404,"data":{},"detailMsg":"No message available","msg":"Not Found"}'

what happend?

**Comment by xysgx** [comment]
I also encountered the same problem. The version of ccxt is 1.18.992
Error display: ccxt.base.errors.exchangenotavailable: post https://www.okex.com/v2/futures/pc/market/futurescoin 404 not found {"code": 404, "data": {}, "detailmsg": "no message available", "MSG": "not found"}

**Comment by kroitor** [comment]
Looks like OKEX has removed the futuresCoin endpoint, because it was not an official endpoint, but a web API interface. The problem is that okex v1 does not allow to get all spot markets and futures markets, therefore we had to use their web API for this. But now it appears to be broken:

- https://github.com/okcoin-okex/API-docs-OKEx.com/issues/38

Obviously, OKEX has changed something on their side. We suggest to migrate to okex v3, since we are going to merge v3 over v1 soon, and the v1 API will not be supported further.

Alternatively, you can turn off the futures functionality for OKEX, like so:

```Python
import ccxt
okex = ccxt.okex({
    'has': {'futures': False},  # ←---------------- switch futures off for okex v1
    'apiKey': 'YOUR_API_KEY',
    'secret': 'YOUR_SECRET',
    # ...,
})

# ...
```

Let us know if the above does not answer the question.

**Comment by kroitor** [comment]
Here's the new URL, they switched it from v2 to v3, leaving everything else intact: https://www.okex.com/v3/futures/pc/market/futuresCoin

**Comment by kroitor** [comment]
This should be fixed in 1.19.35. However, we highly encourage you to migrate to okex v3 asap.

**Comment by JeremyChiaYilin** [comment]
Thanks for your help! @kroitor
